react-frontend/src/components/Items/ShowItem.js

```diff
@@ -20,7 +25,7 @@ class ShowItem extends Component {
     componentDidMount() {
         axios.get(interpolateWithId(APIPaths.items, this.props.match.params.id))
             .then(res => {
-                this.setState({...this.state, item: res.data, loaded: true});
+                this.setState({...this.state, item: res.data, img: res.data.img, loaded: true});
```

Comment:
why not leave this in the item? and get it from there?

Reply:
calling 'this.state.item.img.src' on L65 in the DisplayImage component, it did not have the src data. So the image did not show. Changing it to this.state.img.src worked for some reason.